nexus/db-queries/src/db/datastore/external_ip.rs

```diff
@@ -84,33 +83,14 @@ impl DataStore {
         opctx: &OpContext,
         ip_id: Uuid,
         probe_id: Uuid,
-        pool_name: Option<NameOrId>,
+        pool: Option<authz::IpPool>,
```

Comment:
Mind giving `allocate_instance_snat_ip` similar treatment so we're consistent across everything here

Reply:
Ah yes. I didn’t look hard enough.